release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -141,6 +141,12 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-11-stalld-service-now-delivered-with-OS"]
+==== The stalld service is now delivered with the operating system
+
+The `stalld` service now ships with the operating system instead of the Node Tuning Operator (NTO). With this change, any {product-title} 4.10 cluster that uses the `stalld` service will incur an additional reboot during the 4.10 to 4.11 upgrade. This is one-time issue, and will not occur in any future cluster upgrades.
```

Comment:
The phrasing is ambiguous, and I was super confused at first.  Here is what I read: "stalld now ships instead of the Node Tuning Operator".  After rereading the text a few times, I think what is meant is the following: "stalld now ships with the operator system instead of shipping with the Node Tuning Operator".  
```suggestion
The `stalld` service now ships with the operating system instead of shipping with the Node Tuning Operator (NTO). With this change, any {product-title} 4.10 cluster that uses the `stalld` service will incur an additional reboot during the 4.10 to 4.11 upgrade. This is a one-time issue, and will not occur in any future cluster upgrades.
```

Reply:
Yes, the suggested wording is more precise.  Thank you!